CONTRIBUTING.md

```diff
@@ -8,17 +8,17 @@ By participating in this project, you are expected to uphold our [Code of Conduc
 
 ## Project Documentation
 
-Please consult the [README](./README.md) and [CODEBASE](./CODEBASE.md) files at the root of this repository.
-
+Please consult the [README](./README.md) file at the root of this repository.
+Also see [Project documentation guidelines](./DOCUMENTATION_GUIDELINES.md) if you are interested in improving our documentation.
 ### Bugs
 
 If you find a bug, please open an issue in this repository describing the bug. You can file a bug [here](https://github.com/WordPress/openverse-api/issues/new?template=bug_report.md). You will see a bug report template with the required information you should provide.
 
 After that, don't forget to tag the issue with the "Bug" label.
 
-### Proposing changes or new features
+### Questions or Thoughts?
 
-If you have an idea of a new feature or change to how the Openverse API works, please [start a discussion](https://github.com/WordPress/openverse-api/discussions/new?category=ideas) so we can discuss the possibility of that change or new feature being implemented and released in the future. This lets us come to an agreement about the proposed idea before any work is done.
+Feel free to [join us on Slack](https://make.wordpress.org/chat/) and discuss the project with the engineers and community members on #openverse.
```

Comment:
The "Questions or Thoughts?" section is already present at the bottom of the file. This section should be changed to suggestion opening an issue because we now use issues for discussion as well.